.gitignore

```diff
@@ -35,3 +35,7 @@ vendor
 
 # Log files
 logfile
+
+# Saved app data
+user.json
```

Comment:
This should be (a) configurable and (b) default to `Library` or something. Can be done later but add an issue